crates/swc_ecma_transforms_base/src/resolver/mod.rs

```diff
@@ -400,31 +398,22 @@ impl<'a> VisitMut for Resolver<'a> {
     typed_ref!(visit_mut_ts_intersection_type, TsIntersectionType);
     typed_ref!(visit_mut_ts_type_ref, TsTypeRef);
     typed_decl!(visit_mut_ts_type_param_decl, TsTypeParamDecl);
-    typed!(visit_mut_ts_enum_member, TsEnumMember);
     typed!(visit_mut_ts_fn_param, TsFnParam);
     typed!(visit_mut_ts_indexed_access_type, TsIndexedAccessType);
     typed!(visit_mut_ts_index_signature, TsIndexSignature);
     typed!(visit_mut_ts_interface_body, TsInterfaceBody);
-    typed!(visit_mut_ts_module_ref, TsModuleRef);
     typed!(visit_mut_ts_parenthesized_type, TsParenthesizedType);
     typed!(visit_mut_ts_type_lit, TsTypeLit);
     typed!(visit_mut_ts_type_element, TsTypeElement);
-    typed!(visit_mut_ts_module_block, TsModuleBlock);
-    typed!(visit_mut_ts_namespace_body, TsNamespaceBody);
     typed!(visit_mut_ts_optional_type, TsOptionalType);
-    typed!(visit_mut_ts_param_prop, TsParamProp);
```

Comment:
Removed all these because they're more like expressions that end up in the output instead of types.